packages/terser/src/terser_minified.bzl

```diff
@@ -40,7 +40,7 @@ Can be a .js file, a rule producing .js files as its default output, or a rule p
 
 Note that you can pass multiple files to terser, which it will bundle together.
 If you want to do this, you can pass a filegroup here.""",
-        allow_files = [".js"],
+        allow_files = True,
```

Comment:
why not continue to constrain it? do we no longer have a list of file extensions that should be legal?

Reply:
It could be constrained to `.js` and `.mjs` and `.map` I  suppose and that would be fine. Should I change it to that?

Reply:
I'll change it to that

Reply:
🌮